tests/test_taxi_service.py

```diff
@@ -122,8 +118,6 @@ def setUp(self):
         self.credentials = {"username": "admin.user", "password": "1qazcde3"}
 
     def test_login(self):
-        response = self.client.post(
-            reverse("login"), self.credentials, follow=True
-        )
+        response = self.client.post(reverse("login"), self.credentials, follow=True)
         self.assertTrue(response.context["user"].is_active)
         print(response.context["user"])
```

Comment:
The `print` statement is used for debugging purposes and should be removed from the test case to keep the test output clean.